common/data_refinery_common/test_utils.py

```diff
@@ -8,6 +8,7 @@ class UtilsTestCase(TestCase):
     @patch('data_refinery_common.utils.requests.get')
     def test_get_worker_id_cloud(self, mock_get, mock_get_env_variable):
         """Test that a request is made and the global value is stored"""
+        utils.INSTANCE_ID = None
```

Comment:
What is this change for? It doesn't seem like this change is part of the logging changes, right?

Reply:
I think that I realized that if the test runner ever ran these two tests in the opposite order that it could break them. It is deterministic so they aren't likely to fail until more tests are added, however it's brittle so I decided to fix it while I was thinking about it.

Reply:
Ok - maybe it would be good to put a quick inline comment to explain this, since it is not immediately apparent.